maquetado/scss/_navbar.scss

```diff
@@ -0,0 +1,23 @@
+.nav {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0 0 15px #cfcfcf;
+  display: flex;
+  justify-content: space-between;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 10;
```

Comment:
I think that with a value of `1` or `2` is enough. It's not the best practice to set high `z-index` values